docs/src/pages/_use-hash-param.js

```diff
@@ -0,0 +1,98 @@
+// Adapted from https://github.com/hejmsdz/use-hash-param/blob/3fd6818/src/index.js
```

Comment:
Will this break existing view configs/urls?  If I understand it correctly, the docs will be deployed to the vitessce.io in which urls like `http://vitessce.io/?url=data:,.......` should work but with this new hash param, it seems like they won't quite work

Reply:
For example: https://s3.amazonaws.com/vitessce-data/docs/2021-01-11/d6521c4/demos/index.html/?dataset=vanderbilt&theme=dark doesn't work (because it doesn't use hash?).  Maybe I have this wrong though.  I think we should support both?

Reply:
Ok. I have reverted to only supporting hash params.

Reply:
Sorry, hash or not hash?  I thought the issue was hash.

Reply:
Sorry, I meant to write query params. https://s3.amazonaws.com/vitessce-data/docs/2021-01-13/5366fb3/index.html?dataset=vanderbilt

Reply:
I think we also need to support `index.html/?` style as well because that is what the current website resolves to.  For example `vitessce.io?url=data:,....` -> `vitessce.io/?url=data:,....` so copy and pasting browser urls currently copies the later, but it currently does not work with this demo, I think.

Reply:
When the site is copied to `vitessce.io`, both `vitessce.io/index.html?url=data:,...` and `vitessce.io/?url=data:,...` should work because we have set the root index.html document

![Screen Shot 2021-01-14 at 2 18 46 PM](https://user-images.githubusercontent.com/7525285/104637912-6f17f180-5673-11eb-9f9d-bb089d491ff9.png)

So this is not really an issue on our end, but we can resolve it by using a different host name when accessing the S3 site.

http://vitessce-data.s3-website-us-east-1.amazonaws.com/docs/2021-01-14/5366fb3/?dataset=vanderbilt

The servers at `.s3-website-{region}.amazonaws.com` recognize `index.html` as the default page for subdirectories served over HTTP, while `s3.amazonaws.com` does not (this server works with object "keys" rather than file names, in which the concept of a directory does not really exist) https://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteEndpoints.html